libs9s/s9srpcclient.cpp

```diff
@@ -10588,20 +10587,24 @@ S9sRpcClient::executeRequest(
     request["request_created"] = timeString;
     request["request_id"]      = ++m_priv->m_requestId;
     
-    if (important)
-        printRequestForDebug(request);
+    //printRequestForDebug(request);
 
     while (true)
     {
         S9sString      hostName;
         int            port = 0;
         S9sString      role;
 
-        retval = doExecuteRequest(uri, request);
+        retval = doExecuteRequest(uri, request, redirect);
             
         if (retval && m_priv->m_reply.isRedirect())
             m_priv->rememberRedirect();
 
+        if (!retval || redirect != S9s::AllowRedirect)
```

Comment:
```suggestion
        if (!retval || redirect == S9s::DenyRedirect)
```
I think that testing for true condition is more intuitive.
Some discussion about that can be found [here](https://softwareengineering.stackexchange.com/a/350474)

Reply:
We have more important things like these.

Anyway, I am not even 100 that this will be the only two value in this enum. I have my reason for these small absolutely unimportant details.